server/player/player.go

```
@@ -2059,25 +2074,30 @@ func (p *Player) BreakBlock(pos cube.Pos) {
-		if info.BreakHandler != nil {
-			info.BreakHandler(pos, p.tx, p)
-		}
-		for _, orb := range entity.NewExperienceOrbs(pos.Vec3Centre(), xp) {
-			p.tx.AddEntity(orb)
+	if private {
+		p.ViewPublicBlock(pos)
+		p.s.ViewParticle(pos.Vec3Centre(), particle.BlockBreak{Block: b})
+	} else {
+		p.tx.SetBlock(pos, nil, nil)
+		p.tx.AddParticle(pos.Vec3Centre(), particle.BlockBreak{Block: b})
+		if breakable, ok := b.(block.Breakable); ok  {
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)